README.md

```diff
@@ -40,6 +37,38 @@ To see all the grunt tasks that are available:
 grunt help
 ```
 
+### Using Angular-PatternFly In Your Application
+
+Note:
+
+1. Add Angular-PatternFly as a dependency for your project and you'll receive all the libraries you'll need:
+
+        $ bower install angular-patternfly --save
+
+2. Add the follwing CSS includes to your HTML file(s), adjusting path where needed:
+
+        <!-- Angular-PatternFly Styles -->
+        <link rel="stylesheet" href="bower_components/angular-patternfly/dist/styles/angular-patternfly.min.css" />
+
+3. Add the following script includes to your HTML file(s), adjusting where necessary to pull in only what you need:
+
+        <!-- Angular-PatternFly  -->
+        <script src="bower_components/angular-patternfly/dist/angular-patternfly.min.js"></script>
+
+        <!-- C3, D3 - Charting Libraries -->
+        <script src="bower_components/patternfly/components/c3/c3.min.js"></script>
+        <script src="bower_components/patternfly/components/d3/d3.min.js"></script>
+
+4. (optional) The 'patternfly.charts' module is not a dependency in the default angular 'patternfly' module.
+   In order to use patternfly charts you must add 'patternfly.charts' as a dependency in your application:
+
+        my-app.module.js:
+
+        angular.module('myApp', [
+          'patternfly',
+          'patternfly.charts'
+        ]);
```

Comment:
I'm suggesting we might want to add, adding the patternfly module to the dependencies:

angular.module('myApp', [
         'patternfly'
       ]);

and then state that optionally, add the charts dependency.